htdocs/js/wui.js

```diff
@@ -614,17 +622,26 @@ vz.wui.handleControls = function(action, keepPeriodStartFixed) {
 			break;
 		case 'zoom-in':
 			vz.wui.period = null;
-			if (vz.wui.tmaxnow)
+			if (hover)
+				vz.wui.zoom(hover - delta/4, hover + delta/4);
```

Comment:
Ich habs noch nciht getestet- was passiert wenn das Chart bei tmaxnow steht und relativ weit rechts getoucht wird? Wäre es nicht sinnvoller den Fall mit dem else Zweig unten zu verheiraten?

Reply:
Wenn vz.wui.tamaxnow gesetzt ist (also rechts ist now) gibt es zwei Fälle:

1. Es wird in der rechten hälfte getouched und dann ein zoom-in.
Hat in meiner Implementierung zur folge dass vz.wui.zoom(from, to) mit einem to wert aufgerufen der in der Zukunft liegt (Es sei denn seit den letzen Refresh ist so viel Zeit vergangen dass der Wert in inzwischen in der Vergangenheit liegt.) Dies wird aber von der Funktion zoom so korrigiert dass das Ergebnis das gleiche wäre wie im im zweig von else if (vz.wui.tamaxnow)

2. Es wird in der linken hälfte getouched und dann ein zoom-in.
Hat in meiner Implementierung zur folge dass zentriert zum touchpunkt gezoomt wird. Eine Kombination mit tamaxnow ist hier nicht möglich oder sinnvoll.